CHANGELOG.md

```diff
@@ -31,6 +31,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   that describes them better, and they also behave similar to Mercurial's
   bookmarks.
 
+* `jj unsquash` has been deprecated in favor of `jj squash` and
+  `jj diffedit --restore-descendants`.
+
+  **Rationale:** `jj squash` can do almost everything `jj unsquash` does.
+  For example, `jj squash` can select the paths to be pulled from the source
+  commit to the target commit. To do this with `jj unsquash` requires the use
+  of the interactive mode. For the other cases, `jj diffedit --restore-descendants`
```

Comment:
The third sentence is kinda awkward, but I currently have no idea on how to improve it. ("To do this with ....").

Reply:
I'll try to find a better phrasing.

Reply:
I've rewritten the paragraph completely.